src/type-coercions.md

```diff
@@ -156,6 +156,16 @@ Coercion is allowed between the following types:
 
 * `!` to any `T`
 
+#### Slice DST raw pointer conversions
+
+When `T` and `U` are both "slice DSTs" - ie, slice types or types whose trailing field
+is a slice type - the raw pointer types `*const T`, `*mut T`, `*const U`, and `*mut U`
+encode the number of elements in this slice. Coercions between these raw pointer types
```

Comment:
This is not a coercion: this is a cast. Casts are explicit and use `as`, coercions are implicit.

Reply:
Hmmm yeah looks like I misread the cast docs; I've moved this text to a more appropriate location.